llvm/lib/ExecutionEngine/Orc/MachOPlatform.cpp

```diff
@@ -1175,11 +1175,13 @@ Error MachOPlatform::MachOPlatformPlugin::mergeImageInfoFlags(
 
 Error MachOPlatform::MachOPlatformPlugin::fixTLVSectionsAndEdges(
     jitlink::LinkGraph &G, JITDylib &JD) {
-
+  auto TLVBootStrapSymbolName = G.intern("__tlv_bootstrap");
```

Comment:
We should intern this in the `MachOPlatformPlugin` class itself so that we can re-use it, rather than re-interning it.